content/shmem_malloc.tex

```diff
@@ -54,7 +54,11 @@
     whether an existing allocation is modified and whether new memory is allocated.
     This ensures that all
     \ac{PE}s participate in the memory allocation, and that the memory on other
-    \ac{PE}s can be used as  soon as the local \ac{PE} returns.  The user is
+    \ac{PE}s can be used as  soon as the local \ac{PE} returns.
+    \newtext{The user is responsible for ensuring that no communication
```

Comment:
There should be no particular complications for an app that communicates on a private context simultaneously, for as long as it never targets/use as source the region that gets modified by the shrealloc/shfree; But that text implies this is illegal code. Is this intentional?